app/src/main/java/org/jellyfin/mobile/player/interaction/PlayerNotificationHelper.kt

```diff
@@ -114,6 +114,10 @@ class PlayerNotificationHelper(private val viewModel: PlayerViewModel) : KoinCom
                 }
                 setContentIntent(buildContentIntent())
                 setDeleteIntent(buildDeleteIntent())
+
+                //prevents the notification from being dismissed while playback is ongoing
+                //which causes video to stop. see #1284
```

Comment:
I think it's not necessary to mention the ticket in the code, having it in the PR is enough. I'd also not mention code paths ("video stops") that are never taken after this fix.
```suggestion
```